.github/workflows/utilities-pulumi_state_splitter.yml

```diff
@@ -0,0 +1,22 @@
+on:
```

Comment:
You could also add a name for better UX:
<img width="826" alt="image" src="https://github.com/house-reliability-engineering/pulumi/assets/57622/267fe23a-2bff-4a4c-86c5-89d5e7c5e7df">

```suggestion
name: pulumi_state_splitter tests
on:
```

Reply:
WDYT about a46dec0e3220a5d84986e236d995190888ceeb34 ?